platform/viewer/src/routes/index.tsx

```diff
@@ -68,7 +68,7 @@ const createRoutes = ({
     ...routes,
     ...(showStudyList ? [WorkListRoute] : []),
     ...(customRoutes?.routes || []),
-    ...bakedInRoutes,
+    ...bakedInRoutes.filter(route => route.path !== '/' || showStudyList),
```

Comment:
This is an unnecessary complication IMO. can't we just remove the first index of bakedInRoutes since we already have the WorkListRoute two lines above?

Reply:
You are right... silly me.